package.json

```diff
@@ -10,7 +10,8 @@
     "npm": ">=7.6.0"
   },
   "dependencies": {
-    "fs-extra": "latest"
+    "fs-extra": "latest",
+    "uuid": "^8.3.2"
```

Comment:
Avoid additional dependencies, a weak UUID can probably be created with a single line of JavaScript.
I don't see a reason for a cryptographically strong UUID.

Reply:
I agree that it's a little overkill